client-misk/src/test/kotlin/app/cash/backfila/client/misk/ClientMiskTestingModule.kt

```diff
@@ -48,10 +47,14 @@ internal class ClientMiskTestingModule(
     install(LogCollectorModule())
     install(MiskTestingServiceModule())
     install(EmbeddedBackfilaModule())
-    install(BackfilaModule(
-        BackfilaClientConfig(
-            url = "test.url", slack_channel = "#test"),
-        backfillClasses
-    ))
+    install(
+        BackfilaModule(
+            BackfilaClientConfig(
+                url = "test.url", slack_channel = "#test"
+            )
+        )
+    )
+
+    binder().bindBackfill<TestBackfill>()
```

Comment:
Is this the right place for this though?
Really I only want this to register for the SinglePartition use case.

We want to specify the backfills to register. Either by passing in the classes or adding a function that does the binding.

Reply:
I think it is the right place and we would just have multiple backfill classes for different tests, wdyt?

Reply:
done